cypress.config.js

```diff
@@ -1,18 +1,23 @@
 const { defineConfig } = require('cypress');
 
-const { isProd } = require('./utils/config-utils');
+const ENV_DEV = 'development';
+const ENV_PROD = 'production';
+const isDev = process.env.NODE_ENV === ENV_DEV;
+const isProd = process.env.NODE_ENV === ENV_PROD;
 
 module.exports = defineConfig({
   // https://docs.cypress.io/guides/references/configuration#Global
   env: {
-    NODE_ENV: isProd ? 'production' : 'dev',
+    NODE_ENV: process.env.NODE_ENV || ENV_DEV,
+    DEV: isDev,
+    PROD: isProd,
```

Comment:
What are these used for?

Reply:
We may not need them at all anymore but cypress adds them to the process env which lets us use cy.ENV I think. I’m good to remove them if these still pass.

Reply:
Yeah this is a nice trick. Definitely worth revisiting